Containerfile

```
@@ -31,5 +31,8 @@ RUN cargo build \
 RUN cp target/$(cat /rust_target.txt)/dist/podlet .
 
 FROM scratch
+LABEL org.opencontainers.image.source="https://github.com/containers/podlet"
+LABEL org.opencontainers.image.description="Generate podman quadlet files from a podman command, compose file, or existing object"
```

Except for Podman commands themselves, we try to keep it Uppercased.

```suggestion
LABEL org.opencontainers.image.description="Generate Podman Quadlet files from a podman command, compose file, or existing object"
```